### ISSUE TITLE
Plugin Freezes The Entire UI Moments After Starting PHPStorm

### ISSUE THREAD

**Issue by lots0logs** [swdotcom/swdc-intellij#29]
**Describe the bug**
Plugin causes PHPStorm to freeze within a minute of starting the IDE. This is a new issue that must have started with the most recent update to the plugin.

**ISSUE TYPE**
- Bug Report

**Steps to reproduce**
1. Start PHPStorm
2. Wait ~1 minute

**Screenshots**
N/A

**Desktop (please complete the following information):**
```
=== About ===
Build version: PhpStorm 2020.1.3 Build #PS-201.8538.41 July 8, 2020
Java version: 11.0.7+10-b765.64amd64
Operating System: Linux (5.4.0-42-generic, amd64)
JVM version: OpenJDK 64-Bit Server VM JetBrains s.r.o.
idea.config.path=/home/dustin/.config/JetBrains/PhpStorm2020.1
idea.system.path=/home/dustin/.cache/JetBrains/PhpStorm2020.1
idea.plugins.path=/home/dustin/.local/share/JetBrains/Toolbox/apps/PhpStorm/ch-0/201.8538.41.plugins
idea.log.path=/home/dustin/.cache/JetBrains/PhpStorm2020.1/log

=== System ===
Number of CPU: 24
Used memory: 319Mb 
Free memory: 151Mb 
Total memory: 471Mb 
Maximum available memory: 725Mb

=== Displays ===
Displays: 
Display 0: 2560x1440; scale: 1.00
Display 1: 2560x1440; scale: 1.00

=== Plugins ===
Custom plugins: [Git Branch Cleaner (1.1.4), String Manipulation (8.1.191.000.2), YAML/Ansible support (0.9.5), Switch Component files (1.0), View Package Versions (1.2.0), Material Theme UI (5.3.2.1), Conventional Commit (0.15.1), Shared Indexes (201.8538.6), Lua (1.0.127), OpenResty Lua Support (0.0.4), Ideolog (201.0.22.0), JSON Sorter (1.0.0-beta.3.4), HashiCorp Terraform / HCL language support (0.7.10), React snippets (1.1.0), Theme Collection (0.8), JS Toolbox (1.9), Atom Material Icons (25.1), Code Time (2.2.10), Atom OneDark Theme (1.6), One Dark Native (1.5), nginx Support (0.1.9), Extra Icons (1.32.0.201), Case conversion (0.6.1), Makefile support (3.1.1), Code::Stats (1.0.9), Quick File Preview (1.6.3), CodeGlance (1.5.4), Go Template (201.8538.31.199), Kubernetes (201.7846.6), Node.js Remote Interpreter (201.6668.113), Rainbow Brackets (6.7), .env files support (2020.3), deep-assoc-completion (2020.06.26.001), Php Inspections (EA Extended) (4.0.5), Puppet (201.8538.45)]
Disabled plugins:[One Dark theme (5.0.0), CodeStream (8.1.3+74), PhpStorm Workshop (201.8538.41), Space (2020.1.4), Find Pull Request (1.7.1), DQL (201.8538.41), PHPT Support (201.8538.41), Joomla! (201.8538.41), Settings Repository (201.8538.41), Perforce (201.8538.41), Vue.js (201.8538.41), ASP (201.8538.41), Subversion (201.8538.41), CoffeeScript (201.8538.41), Gray Theme (1.1.1), Vagrant (201.8538.41), Mercurial (201.8538.41), Haml (201.8538.41), Drupal (201.8538.41), Angular and AngularJS (201.8538.41)]
```

**Additional context**
```
"AWT-EventQueue-0" prio=0 tid=0x0 nid=0x0 blocked
     java.lang.Thread.State: BLOCKED
 on java.util.concurrent.Semaphore@227d12b8 owned by "pool-2-thread-1" Id=97
	at com.softwareco.intellij.plugin.managers.FileManager.getFileContent(FileManager.java:561)
	at com.softwareco.intellij.plugin.managers.FileManager.getJsonObjectFromFile(FileManager.java:526)
	at com.softwareco.intellij.plugin.managers.FileManager.getSoftwareSessionAsJson(FileManager.java:521)
	at com.softwareco.intellij.plugin.managers.FileManager.getItem(FileManager.java:472)
	at com.softwareco.intellij.plugin.SoftwareCoStatusBarKpmTextWidget.setTooltip(SoftwareCoStatusBarKpmTextWidget.java:43)
	at com.softwareco.intellij.plugin.SoftwareCoUtils$1.run(SoftwareCoUtils.java:410)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:201)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:831)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:310)
	at com.intellij.openapi.application.impl.ApplicationImpl$$Lambda$299/0x0000000100489840.run(Unknown Source)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:80)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:128)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:46)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:184)
	at java.desktop@11.0.7/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop@11.0.7/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop@11.0.7/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop@11.0.7/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base@11.0.7/java.security.AccessController.doPrivileged(Native Method)
	at java.base@11.0.7/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop@11.0.7/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:974)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:847)
	at com.intellij.ide.IdeEventQueue.lambda$null$8(IdeEventQueue.java:449)
	at com.intellij.ide.IdeEventQueue$$Lambda$639/0x00000001008dd840.compute(Unknown Source)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:741)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
	at com.intellij.ide.IdeEventQueue$$Lambda$636/0x00000001008df040.run(Unknown Source)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:831)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:496)
	at java.desktop@11.0.7/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop@11.0.7/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop@11.0.7/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop@11.0.7/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop@11.0.7/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop@11.0.7/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by mmclead** [comment]
Thanks for the error report.   The stack trace is very helpful.    We'll take a look and get an update as soon as we can.

Once we have a fix there is a 1-2 day delay for approval by Jetbrains to approve in their marketplace

**Comment by tolotrasamuel** [comment]
Hey! What's the immediate workaround??? I cannot code anymore right now because PhpStorm freezes completely! My project is on hold, please provide steps to revert this

**Comment by tolotrasamuel** [comment]
I cannot even delete the plugin because PhpStorm won't even start up !!

**Comment by xavluiz** [comment]
Here's a link to uninstall without booting up. We have a PR ready and will get this updated as soon as the review completes, thank you.

https://intellij-support.jetbrains.com/hc/en-us/articles/206544519

**Comment by xavluiz** [comment]
Jetbrains has approved the update and the fix for this now available in v2.2.11. We apologize for the inconvenience.

Regards.
